git.zsh

```diff
@@ -72,14 +148,26 @@ _fzf_complete_git-commits_post() {
 _fzf_complete_git-commit-messages() {
     local fzf_options="$1"
     shift
-    _fzf_complete "--ansi --tiebreak=index $fzf_options" "$@" < <(git log --color=always --format='%C(yellow)%h%C(reset)  %s' 2> /dev/null)
+    _fzf_complete "--ansi --tiebreak=index $fzf_options" "$@" < <(
+        git log --format='%h %s' 2> /dev/null |
+        awk -v prefix="$prefix_option" '
+            {
+                match($0, / /)
+                print $1, prefix substr($0, RSTART + RLENGTH)
+            }
+        ' |
+        _fzf_complete_git_tabularize
+    )
 }
 
 _fzf_complete_git-commit-messages_post() {
-    awk '{
-        $1 = ""
-        sub(/^ /, "", $0)
-        print "'\''" $0 "'\''"
+    awk -v prefix="$prefix_option" '{
+        match($0, /  /)
+        str = substr($0, RSTART + RLENGTH)
+        match(str, prefix)
+        str = substr(str, RSTART + RLENGTH)
+        gsub("'\''", "'\''\\'\'''\''", str)
+        print prefix "'\''" str "'\''"
```

Comment:
This should be a convenient function that can be generally used in other awk invocations. Can you add it to `_fzf_complete_awk_functions `?